02-regex/README.md

```diff
@@ -73,6 +73,7 @@ In preparation for next week, add a link to a data source or API (even just data
 - Shun -- [2018 Central Park Squirrel Census Fur Color Map](https://data.cityofnewyork.us/Environment/2018-Squirrel-Census-Fur-Color-Map/fak5-wcft)
 - Wallis -- [WordNet](https://wordnet.princeton.edu/)
 - Joyce -- [Google Maps Javascript API](https://developers.google.com/maps/documentation/javascript)
+- Lening --[https://repeated-cake-04e.notion.site/Week2-6bfc3115ae624586a4238f39fbb8317e?pvs=4]
```

Comment:
Did you mean for this link to be with the data section? I think this is your blog post which can go below!